scripts/skill_auditor/README.md

```
@@ -0,0 +1,68 @@
+
+```bash
+# Audit a skill
+./scripts/skill-auditor.py /path/to/skill/directory
+
+# Example
+./scripts/skill-auditor.py plugins/meta/meta-claude/skills/skill-factory
+```
+
+## What It Checks
+
+### Blockers (Must Fix)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)